opennms-config-model/src/main/java/org/opennms/netmgt/config/eventd/EventdConfiguration.java

```diff
@@ -127,6 +127,36 @@
 
     @XmlValue
     private String m_contents;
+    
+    
+    /**
+     * Number of threads used for consuming/dispatching messages.
+     * Defaults to 2 x the number of available processors.
+     */
+    @XmlAttribute(name = "sink-threads", required=false)
```

Comment:
Can we prefix **all** the sink related settings we added with the 'sink-' prefix?